cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -132,6 +132,9 @@ public TestCommand() {
     @CommandLine.Option(names = "--debug", description = "start in remote debugging mode")
     private String debugPort;
 
+    @CommandLine.Option(names = "--workers", description = "maximum number of parallel test jobs", defaultValue = "1")
```

Comment:
IMO the wording `test jobs` is vague.
Perhaps we can use something like,
```suggestion
    @CommandLine.Option(names = "--workers", description = "number of threads allocated for test execution", defaultValue = "1")
```

Reply:
We decided to remove `--workers` option completely and introduce `--parallel` option